LICENSE-DETAILS.md

```diff
@@ -0,0 +1,80 @@
+# Detailed Licensing Information
+
+This crate started as a port of LLVM's APFloat and APInt code (revision [`f3598e8fca83ccfb11f58ec7957c229e349765e3`](https://github.com/llvm/llvm-project/commit/f3598e8fca83ccfb11f58ec7957c229e349765e3)).
```

Comment:
```suggestion
This crate started as a port of LLVM's APFloat and (parts of) APInt code (revision [`f3598e8fca83ccfb11f58ec7957c229e349765e3`](https://github.com/llvm/llvm-project/commit/f3598e8fca83ccfb11f58ec7957c229e349765e3)).
```
(is this a good idea? I'm not sure how to express something like a "just a small part of" for `APInt`, vs `APFloat` where (almost) everything was ported)

Reply:
This seems fine to me, it's more accurate and alludes to why we don't need to consider portions of https://github.com/llvm/llvm-project/commit/1c419ff50d35c4cab442f5a1c8f5e82812921633 as relevant for this discussion.